packages/stark-ui/assets/styles/_material-fixes.scss

```diff
@@ -19,3 +19,7 @@
     display: block;
   }
 }
+
+.mat-button-wrapper .mat-icon {
```

Comment:
Can you add a comment to explain since which version of Angular Material is this fix needed?